### PR TITLE
fix(tile): adjust the heading styling to be word wrapped including long string texts #3215

### PR DESCRIPTION
**Pull request by y0n4**
**Related Issue:** #3215

## Summary
`calcite-tile` heading text will word wrap long string texts. It will also break the word halfway if the a single string is too long for the width

Screen Shot 2021-11-04 at 10 19 21 AM

### CODE REVIEW

File: src/components/calcite-tile/calcite-tile.scss [hunk 1]
```diff
@@ -77,6 +79,9 @@
     ease-in-out
     font-medium
     pointer-events-none;
+  white-space: nowrap;
```
Comment (driskull) [inline src/components/calcite-tile/calcite-tile.scss:82]: Can we use tailwind CSS where possible?

**Review by driskull — COMMENTED** [review]
just a comment :)

**Review by driskull — APPROVED** [review]
Looks good to me! Is a screener test needed to make sure this doesn't break?

**Comment by y0n4** [comment]
Ah, good point thanks. I will adjust the story for the screener test to catch that

**Pull request merged**